backends/p4tools/modules/testgen/CMakeLists.txt

```diff
@@ -114,7 +114,9 @@ target_link_libraries(
 add_custom_target(
   linkp4testgen
   # Add some convenience links for invoking p4testgen.
+  # TODO: How to best handle includes?
   COMMAND ${CMAKE_COMMAND} -E create_symlink ${CMAKE_CURRENT_BINARY_DIR}/p4testgen ${CMAKE_BINARY_DIR}/p4testgen
+  COMMAND ${CMAKE_COMMAND} -E create_symlink ${P4C_BINARY_DIR_PATH_REL}/p4include ${CMAKE_CURRENT_BINARY_DIR}/p4include
```

Comment:
Interestingly, this does not work for me (maybe because I have p4c embedded in other project, although it worked the recent testgen changes...). The creates symlink leads to `/p4include`, suggesting that `${P4C_BINARY_DIR_PATH_REL}` is not set. As far as I can see, the variable is set in `ebfp`, `bmv2` and `p4test` backends, but not in `p4tools`. It would probably need to be set in this file to work correctly.